.github/workflows/publish.yaml

```diff
@@ -16,7 +16,9 @@ jobs:
       - id: set
         name: Set OLM Tag
         run: |
-          olm_tag=$(yq .spec.install.spec.deployments[0].spec.template.spec.containers[0].image bundles/cilium.v${{ github.event.inputs.version }}/manifests/cilium.clusterserviceversion.yaml | cut -d ':' -f 2)
+          VERSION="${{ github.event.inputs.version }}"
+          set -o pipefail
+          olm_tag=$(yq .spec.install.spec.deployments[0].spec.template.spec.containers[0].image bundles/cilium.v${VERSION#v}/manifests/cilium.clusterserviceversion.yaml | cut -d ':' -f 2)
```

Comment:
```
${VERSION#v}
```
what does this do? 😇

Reply:
@michi-covalent: It removes the shortest matching prefix pattern (in our case, 0 or 1 prefixing `v`) from the variable.

```bash
$ foo=1.99
$ echo ${foo#v}
1.99
$ foo=v1.99
$ echo ${foo#v}
1.99
```

From `man bash`:

```
${parameter#word}
${parameter##word}
       Remove  matching prefix pattern.  The word is expanded to produce a pattern
       just as in pathname expansion, and matched against the  expanded  value  of
       parameter  using  the rules described under Pattern Matching below.  If the
       pattern matches the beginning of the value of parameter, then the result of
       the expansion is the expanded value of parameter with the shortest matching
       pattern (the ``#'' case) or the longest matching pattern (the ``##''  case)
       deleted.   If parameter is @ or *, the pattern removal operation is applied
       to each positional parameter in turn, and the expansion  is  the  resultant
       list.   If parameter is an array variable subscripted with @ or *, the pat‐
       tern removal operation is applied to each member of the array in turn,  and
       the expansion is the resultant list.
```

Reply:
fascinating ❤️